DEVELOPERS.md

```diff
@@ -214,6 +217,24 @@ Vite builds the assets and outputs them to the `assets/dist` folder.
 
 [Django Staticfiles app](https://docs.djangoproject.com/en/3.2/ref/contrib/staticfiles/) then collects the files and places them in the `staticfiles/assets` folder, with the manifest file located at `staticfiles/manifest.json`.
 
+### Data Setup
+Sometimes it's useful to have a fresh local installation or you may not have authorization to download a production backup.
+In that situation you can follow the steps below to set up your local copy of the site:
+
+1. Create [a GitHub OAuth application](https://github.com/settings/applications/new), the fields don't matter too much for local dev, except the callback URL which needs to be `http://localhost:8000/complete/github/`.  Then set the `SOCIAL_AUTH_GITHUB_KEY` and `SOCIAL_AUTH_GITHUB_KEY` environment variables with values from that application.
+1. Register a user account on your local version of job-server by clicking Login
+1. Give your user the `CoreDeveloper` role by setting the `ADMIN_USERS` environment variable to include your username, then running `python manage.py ensure_admins`
```

Comment:
Similar to https://github.com/opensafely-core/job-server/pull/1795/files#r870188778

```suggestion
1. Give your user the `CoreDeveloper` role by:
   1. Setting the `ADMIN_USERS` environment variable to include your username.
   1. Running `python manage.py ensure_admins`
```

(Same caveat about list formatting as https://github.com/opensafely-core/job-server/pull/1795/files#r870188778 too.)